ovh/types_dedicated_server.go

```diff
@@ -111,24 +111,39 @@ type DedicatedServerInstallTaskCreateOpts struct {
 	TemplateName        string                             `json:"templateName"`
 	PartitionSchemeName *string                            `json:"partitionSchemeName,omitempty"`
 	Details             *DedicatedServerInstallTaskDetails `json:"details"`
+	UserMetadata        *[]map[string]string               `json:"userMetadata"`
```

Comment:
- you should have an array of a struct instead of a map since we know the keys
- the pointer seems useless
- you should add an `omitempty` ?